tools/operator/launcher/k8s/connprofile.go

```diff
@@ -32,6 +32,7 @@ func init() {
 	}
 }
 
+//K8Ports --
```

Comment:
I know you may not keep these exported, but for the semantic parser, there is a space between the slashes and the comment 
```
// K8Ports --
```